staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/controller.go

```diff
@@ -99,7 +99,13 @@ func (c *AggregationController) processNextWorkItem() bool {
 		return false
 	}
 
-	klog.Infof("OpenAPI AggregationController: Processing item %s", key)
+	if aggregator.IsLocalAPIService(key.(string)) {
+		// for local delegation targets that are aggregated once per second, log at
+		// higher level to avoid flooding the log
+		klog.V(4).Infof("OpenAPI AggregationController: Processing item %s", key)
```

Comment:
I'm not sure if `V(4)` is appropriate. maybe we can go with higher level? do we have verbose level convention documented somewhere?

Reply:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md

this seems like v(5) to me

Reply:
@liggitt thanks! I updated the number